exercises/card_counting.livemd

```diff
@@ -176,7 +181,21 @@ You could omit the **normal** cards or allow **high** and **low** cards to balan
 Enter your solution below.
 
 ```elixir
+count = 0
+
+[card1, card2, card3] = [Enum.random(1..13), Enum.random(1..13), Enum.random(1..13)]
+
+card_list = [card1, card2, card3]
+
+Enum.reduce(card_list, 0, fn card, acc ->
+  IO.inspect(card)
 
+  cond do
+    card < 7 -> acc = acc - 1
```

Comment:
We encountered this today. Remember that we don't rebind the accumulator, the next accumulator will be the return value of our function.

The warning `warning: variable "acc" is unused (there is a variable with the same name in the context, use the pin operator (^) to match on it or prefix this variable with underscore if it is not meant to be used)
  Documents/GitHub/beta_curriculum/exercises/card_counting.livemd#cell:9` occurs because we have bound `acc = acc` in the return value of our `cond` statement. We don't then use the bound variable, so Elixir let's us know we are probably doing something wrong.

Try changing the cond statement to resolve the issue.

 ```elixir
 cond do
   card < 7 -> acc + 1
   card > 9 -> acc - 1
   true -> acc
 end
 ```

Reply:
Resolved! Thank you :)